docs/class-definitions.md

```diff
@@ -224,10 +224,10 @@ space.members.unsubscribe('leave');
 
 #### getSelf()
 
-Returns a Promise which resolves to the [SpaceMember](#spacemember) object relating to the local connection. Will resolve to `undefined` if the client hasn't entered the space yet.
+Returns a Promise which resolves to the [SpaceMember](#spacemember) object relating to the local connection. Will resolve to `null` if the client hasn't entered the space yet.
 
 ```ts
-type getSelf = () => Promise<SpaceMember | undefined>;
+type getSelf = () => Promise<SpaceMember | null>;
```

Comment:
I made a separate PR to make sure we always return `null` from these `get*` methods https://github.com/ably-labs/spaces/pull/141